examples/paywall-tester/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+}
+
+apply from: "$rootProject.projectDir/base-application.gradle"
+
+android {
+    namespace 'com.revenuecat.paywallstester'
+
+    defaultConfig {
+        applicationId "com.revenuecat.purchases_sample"
+        minSdk 21
+        versionCode 1
+        versionName "1.0"
+
+        // Library modules have a dimension used to separate different apis.
+        // Our applications however don't need the extra flavor. This makes sure that we use the
+        // default flavor.
+        missingDimensionStrategy 'apis', 'defaults'
+
+        vectorDrawables {
+            useSupportLibrary true
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.4.0-alpha02'
```

Comment:
Yes, according to https://developer.android.com/jetpack/androidx/releases/compose-kotlin, since we are using kotlin 1.8.0, we need to use `1.4.0-alpha02`. We could update to Kotlin 1.8.20 though... It just seems relatively recent. Thoughts on that?

Reply:
nah let's keep it and don't introduce more breaking points